src/components/cc-tile-requests/cc-tile-requests.js

```diff
@@ -72,21 +72,8 @@ export class CcTileRequests extends withResizeObserver(LitElement) {
 
     /** @type {boolean} */
     this._skeleton = false;
-  }
 
-  /** @protected */
-  onResize ({ width }) {
-    if (width < 380) {
-      this._barCount = 6;
-    }
-    if (width >= 380 && width < 540) {
-      this._barCount = 8;
-    }
-    if (width >= 540) {
-      this._barCount = 12;
-    }
-    this._refreshChart();
-    this.requestUpdate();
+    new ResizeController(this);
```

Comment:
issue: let's keep a reference to the controller so we can get its `width` property

```js
this._resizeController = new ResizeController(this);
```